koku/sources/storage.py

```diff
@@ -150,6 +151,17 @@ def load_providers_to_delete():
     return providers_to_delete
 
 
+def get_source(source_id, err_msg):
```

Comment:
What type is source_id, and err_msg?
Can we use type hinting here? https://docs.python.org/3/library/typing.html

Reply:
I'm not entirely sure I like the idea of type hinting. Hinting is certainly nice for a library, but we aren't building one.

I can add to the function docstring though.

Reply:
👍